### DISCUSSION TITLE
Foundry Fundamentals

### DISCUSSION THREAD

**Discussion by orange810-ui** [Cyfrin/foundry-full-course-cu#3037] [General]
i got this error while runing this command
 "forge test " or " forge install " in terminal 
""Error: failed to extract foundry config: foundry config error: data did not match any variant of untagged enum SoldeerDependencyValue for setting `dependencies.Chainlink`"" how i fix this

**Comment by EngrPips** [comment]
Hello, please show your `Foundry.toml` file.

**Reply by orange810-ui** [reply]
this is my Foundry.toml file
'''
[profile.default]
src = "src"
out = "out"
libs = ["lib"]
remappings = [
    "@chainlink/contracts=lib/chainlink/contracts"
]

[dependencies] 
Chainlink = { git = "https://github.com/smartcontractkit/chainlink.git", tag = "v0.8.0" }
'''

**Comment by EngrPips** [comment]
Where did you get the below from?

```js
[dependencies]
Chainlink = { git = "https://github.com/smartcontractkit/chainlink.git", tag = "v0.8.0" }
``` 

Did Patrick tell you to put it there?

**Reply by orange810-ui** [reply]
No, but i tried that one which told by patrick , and also i retried all those steps which told by patrick but error was showing after many tries and after do all tries i told all problems to chat gpt then i found this solution but i see same error........so can you help me?

**Reply by EngrPips** [reply]
Please remove that `dependencies` stuff, retry running those commands and show me whatever error you get.

**Reply by orange810-ui** [reply]
I share a screen shot with you after removing that dependencies.............

![Screenshot (374)](https://github.com/user-attachments/assets/7d7e8850-93f1-4d08-b6d0-3e5ff8178c25)

**Comment by orange810-ui** [comment]
Thank you, you did not give me any solution but after telling you the problem I myself understood the problem in a better way or I also solved it, so again I am thankful to you if i did not tell you, maybe I could not find the solution, thank you. 

![Screenshot (376)](https://github.com/user-attachments/assets/93da3482-b217-41f4-9779-80ab30a8e7e3)

**Reply by EngrPips** [reply]
Awesome, I am glad you figured it out. Keep crushing it.

**Comment by Hamobcdev** [comment]
thanks guys for this especially @EngrPips  yeah i had this problem since yesterday and by removing the dependencies also from chat gpt code generated, build successful. Cheers and thank you both.

**Reply by EngrPips** [reply]
Anytime, Friend. Keep crushing it.
